package.json

```diff
@@ -21,9 +21,7 @@
     "test:match": "cd packages/astro && pnpm run test:match",
     "test:unit": "cd packages/astro && pnpm run test:unit",
     "test:unit:match": "cd packages/astro && pnpm run test:unit:match",
-    "test:smoke": "pnpm test:smoke:example && pnpm test:smoke:docs",
-    "test:smoke:example": "turbo run build --concurrency=100% --filter=\"@example/*\"",
-    "test:smoke:docs": "turbo run build --filter=docs",
```

Comment:
Same as my other comment. The smoke test is fine, it's just one step of the test.